budget-app/src/main/webapp/recordSpending.html

```diff
@@ -62,4 +63,71 @@ <h1>Record your spending</h1>
     </div>
     <script src="recordSpending.js"></script>
   </body>
+=======
+.dropdown-content {
+  display: none;
+  position: absolute;
+  background-color: #f1f1f1;
+  min-width: 160px;
+  overflow: auto;
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+  z-index: 1;
+}
+
+.dropdown-content a {
+  color: black;
+  padding: 12px 16px;
+  text-decoration: none;
+  display: block;
+}
+
+.dropdown a:hover {background-color: #ddd;}
+
+.show {display: block;}
+</style>
+</head>
+<body>
+<h1>Record your spending</h1>
+
+<div class="dropdown">
+  <button onclick="myFunction()" class="dropbtn">Choose your Budget Type</button>
+  <div id="myDropdown" class="dropdown-content">
+    <p>FOOD</p>
+    <a >CLOTHING</a>
+    <a >TRANSPORTATION</a>
+    <a >OTHER</a>
+  </div>
+</div>
+
+<form method="POST">
+    <p>How much you spent?</p>
+    <textarea name="text-input"> </textarea>
+    <br/>
+    <input type="submit" />
+  </form>
+
+<script>
+/* When the user clicks on the button, 
+toggle between hiding and showing the dropdown content */
+function myFunction() {
+  document.getElementById("myDropdown").classList.toggle("show");
+}
+
+// Close the dropdown if the user clicks outside of it
+window.onclick = function(event) {
+  if (!event.target.matches('.dropbtn')) {
+    var dropdowns = document.getElementsByClassName("dropdown-content");
+    var i;
+    for (i = 0; i < dropdowns.length; i++) {
+      var openDropdown = dropdowns[i];
+      if (openDropdown.classList.contains('show')) {
+        openDropdown.classList.remove('show');
+      }
+    }
+  }
+}
+</script>
+
+</body>
+>>>>>>> 53acedc33b6e7c0c684c7b0dd0f2809cb79f7253
```

Comment:
And lastly this is where the merge conflict ends. Please remove this line as well.